NBitcoin/RPC/RPCClient.Wallet.cs

```diff
@@ -858,6 +861,21 @@ public async Task<SignRawTransactionResponse> SignRawTransactionWithWalletAsync(
 			return response;
 		}
 
+		public WalletProcessPSBTResponse WalletProcessPSBT(PSBT psbt, bool sign = true, SigHash hashType = SigHash.All, bool bip32derivs = false)
+			 => WalletProcessPSBTAsync(psbt, sign, hashType, bip32derivs).GetAwaiter().GetResult();
+		public async Task<WalletProcessPSBTResponse> WalletProcessPSBTAsync(PSBT psbt, bool sign = true, SigHash sighashType = SigHash.All, bool bip32derivs = false)
+		{
+			if (psbt == null)
+				throw new ArgumentNullException(nameof(psbt));
+
+			var response = await SendCommandAsync(RPCOperations.walletprocesspsbt, psbt.ToBase64(), sign, SigHashToString(sighashType), bip32derivs ).ConfigureAwait(false);
+			var result = response.Result as JObject;
```

Comment:
if you're not comparing "result" against null later, you should use a normal (static) cast instead of the keyword "as" (dynamic cast)